src/_guides/language/concurrency/index.md

```diff
@@ -462,6 +458,42 @@ is slower when isolates are in different groups.
   Flutter doesn't support `Isolate.spawnUri()`.
 {{site.alert.end}}
 
+## Concurrency on the web
+
+All Dart apps can use `async-await`, `Future`, and `Stream`.
+Isolates, however, are not available on the [Dart web platform][].
+Dart web apps can use [web workers][] to
+run scripts in background threads
+similar to isolates.
+Web workers' functionality and capabilities
+differ somewhat from isolates, though.
+
+For instance, when web workers send data between threads,
+they copy the data back and forth.
+Data copying can be very slow, though,
+especially for large messages. 
+So, rather than copying, isolates _transfer_ the memory
+that holds the message in the exiting isolate
+to the receiving isolate
+(specifically by `Isolate.exit()`).
+
+{% comment %}
+Add ", and by extension `Isolate.run()`" to parenthesis after 2.19 and `run()` releases.
+{% endcomment %}
+
+How you create web workers and isolates also differs.
+On the web, you can only create web workers by declaring
+a separate program entrypoint and compiling that separately.
+Natively, you can start isolates in a similar manner,
+with separate sources, separate from existing isolates.
+More typically though, creation means branching from an existing isolate.
```

Comment:
**suggestion:** Tighten and active voice. This builds on @lrhn 's revision.
```suggestion
The method to create web workers and isolates differs.
To create web workers, declare a separate program entry point
and compile that code separately.
A web worker starts a similar way to starting an isolate using `Isolate.spawnUri`.
This method uses more resources than using `Isolate.spawn`.
The latter method [can reuse some of the same code and data](#performance-and-isolate-groups) as the spawning isolate.
```

Reply:
> To create web workers, declare a separate program entry point
and compile that code separately.

"declare" is odd to me here. I feel like explaining a concept is decidedly not active, if that makes sense. We're not trying to literally get them to declare an entry point right now. I know always active is technically the rule in general but there has to be a caveat in this kind of situation?

Reply:
**reply:** If you think it reads better as is. That's basically the caveat.